langchain/document_loaders/web_base.py

```diff
@@ -110,7 +110,7 @@ async def _fetch(
             for i in range(retries):
                 try:
                     async with session.get(
-                        url, headers=self.session.headers, verify=self.verify
```

Comment:
seems like there is a verify_ssl kwarg, could that be what this is supposed to be?

Reply:
Yes.

From aiohttp docs:
verify_ssl – Perform SSL certificate validation for HTTPS requests (enabled by default). May be disabled to skip validation for sites with invalid certificates.

This bug is also impacting WebBaseLoader when calling async, fyi.

Reply:
@dev2049 @pandemosth 

`verify_ssl` arg has been deprecated, and replaced with:

ssl –
SSL validation mode. None for default SSL check
([ssl.create_default_context()](https://docs.python.org/3/library/ssl.html#ssl.create_default_context) is used), False for skip SSL certificate validation, [aiohttp.Fingerprint](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.Fingerprint) for fingerprint validation, [ssl.SSLContext](https://docs.python.org/3/library/ssl.html#ssl.SSLContext) for custom SSL certificate validation.
Supersedes verify_ssl, ssl_context and fingerprint parameters.
(From aiohttp docs: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientSession.request)

I've updated the code to use `ssl` param